aapp_runner/tests/test_helper_functions.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+
+# Copyright (c) 2022 Pytroll developers
+
+# Author(s):
+
+#   Adam Dybbroe <Firstname.Lastname @ smhi.se>
+
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+"""Unittesting the helper functions for the AAPP-runner.
+"""
+
+import logging
+import unittest
+from datetime import datetime
+from unittest.mock import patch
+
+from aapp_runner.helper_functions import check_if_scene_is_unique
+from aapp_runner.read_aapp_config import AappL1Config, AappRunnerConfig
+from aapp_runner.tests.test_config import (TEST_YAML_CONTENT_OK,
+                                           create_config_from_yaml)
+
+
+class TestProcessConfigChecking(unittest.TestCase):
```

Comment:
Should we use pytest rather than unittest.TestCase, so we can use the more powerful functionality it provides?

Reply:
Probably. I wanted to use the `self.assertLogs`, and didn't know how to do that with pytest. But perhaps we can leave it, as it works fine for the purpose needed?